pkg/controllers/repositoryserver/testutils.go

```diff
@@ -57,3 +63,95 @@ func getDefaultS3CompliantStorageLocation() map[string][]byte {
 		reposerver.EndpointKey: []byte(os.Getenv("LOCATION_ENDPOINT")),
 	}
 }
+
+func createKopiaRepository(cli kubernetes.Interface, rs *v1alpha1.RepositoryServer, storageLocation map[string][]byte) error {
```

Comment:
I don't see this being used in this PR. Why do we need it in this PR?

Reply:
While rebasing the PR last time, few tests got deleted which was using this function. I have added them back.